.travis.yml

```
@@ -10,9 +9,17 @@ matrix:
       sudo: required
   allow_failures:
     - python: "nightly"
-  
+
 # command to install dependencies
-install: 
-  - pip install pytest mock
+install:
+  - |
+    pip install kodi-addon-checker
+    pip install pytest mock
```

No need to make this a single 'command'.
```suggestion
install:
    - pip install kodi-addon-checker
    - pip install pytest mock
```